lib/a_star/Makefile

```diff
@@ -0,0 +1,25 @@
+# This file is part of NIT ( http://www.nitlanguage.org ).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+NITUNIT ?= nitunit
+NITDOC ?= nitdoc
+
+check:
+	$(NITUNIT) .
+
+doc:
+	$(NITDOC) . -o doc/
+
+clean:
+	rm -rf doc/
```

Comment:
I'm not sure that a Makefile is required for lib-only packages. This seems to add more noise that signal. But maybe I miss something.